en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md

```diff
@@ -566,6 +603,15 @@ At any given time, the state of the endpoint can be one of the following. During
       </tr>
    </thead>
    <tbody>
+      <tr>
+         <td>
+            <p>Timeout Duration</p>
+         </td>
+         <td>
+            Connection timeout interval. If the remote endpoint does not respond in this time, it will be marked as "Timeout." This can be in defined as a static value or as a dynamic value.</br>
```

Comment:
```suggestion
            Connection timeout interval. If the remote endpoint does not respond in this time, it will be marked as "Timeout." This can be defined as a static value or as a dynamic value.</br>
```